src/components/line-chart-audification/line-chart-audification.component.spec.ts

```diff
@@ -106,6 +102,14 @@ describe('LineChartAudificationComponent', () => {
     }
   });
 
+  it('should switch the legend item upon pressing UP or DOWN.', () => {
+    ['ArrowUp', 'ArrowDown'].forEach(key => {
+      const prevDatumIndex = component.datumIndex;
+      triggerKeyDown(key);
+      expect(component.datumIndex).not.toBe(prevDatumIndex);
```

Comment:
should you also be checking that the legend item changes focus?

Reply:
Which element do you mean by the legend item?

Reply:
The test describes checking that UP/DOWN changes the legend item, but it looks like the test just verifies that new index is not the old index. Should it maybe verify that the new index is the correct to focus?